test/kernels/gibbskernel.jl

```diff
@@ -0,0 +1,12 @@
+@testset "gibbskernel" begin
+    x = randn()
+    y = randn()
+
+    # this is the gibbs lengthscale function.
+    ell(x) = exp(sum(sin, x))
+    # create a gibbs kernel with our specific lengthscale function
+    k_gibbs = GibbsKernel(ell)
+
+    @test k_gibbs(x, y) ≈ sqrt((2 * ell(x) * ell(y)) / (ell(x)^2 + ell(y)^2)) * exp(- norm(x - y)^2 / (ell(x)^2 + ell(y)^2))
```

Comment:
No need for `norm` here:
```suggestion
    @test k_gibbs(x, y) ≈ sqrt((2 * ell(x) * ell(y)) / (ell(x)^2 + ell(y)^2)) * exp(-(x - y)^2 / (ell(x)^2 + ell(y)^2))
```

Reply:
Ah ok - i suppose if we used something like `x=rand(2)` or something then we should use `norm` ?

Reply:
Yes, if `x` and `y` would be vectors it would be needed.